Feature/mobile menu

Roathy opened the pull request:
In this branch I created the html for the navigation menu for the mobile version, add the styles to it and implement js for the event and display.

✅ No linters errors
✅ Navigation menu
✅ Click or tap on hamburger makes menu appear
✅ Click or tap on x makes menu disappear
✅ Click on option takes to that part of the website

Roathy committed:
Configure HTML-CSS-JS linters

Roathy committed:
Feature navbar menu for mobile version

Roathy committed:
Add classes for styling and show/hide menu on click event

Roathy committed:
Set liveserver port

Roathy committed:
Fix stylelint errors

Roathy committed:
Add stylelint.json file

Roathy committed:
Add scripts.js file

Salarhabibi-amir committed:
Update README.md

AdnanOlarmmi commented on scripts.js:
- [ ] Kindly note that the requirement is that, the nav bar item is not visible on mobile screen but the hamburger menu button is not visible on desktop size, but it is visible on mobile devices. Upon clicking or tapping the hamburger button, the mobile menu will appear. To close the mobile menu, the user can click or tap the close (X) button. Selecting any option from the mobile menu will display the corresponding part of the page, and the mobile menu will disappear upon selecting an option. The design of the hamburger button and menu adheres to the chosen template in Figma, including elements such as font, colors, images, text, and margins.

- This is what your design currently looks like.
- ![mobile](https://github.com/Roathy/-Portfolio-/assets/41163571/da8c436c-3804-4c84-a735-6bbbe03d3ce6)

AdnanOlarmmi commented on index.html:
```diff
@@ -329,7 +340,15 @@ <h2>Contact me</h2>
             </div>
         </section>
     </div>
+    <script>
+        function openNav() {
+            document.getElementById("myNav").style.display = "block";
+        }
 
+        function closeNav() {
+            document.getElementById("myNav").style.display = "none";
+        }
+    </script>
```
- [ ] It's also not a good practice for you javascript code to be inside the HTML,

Roathy committed:
Add mobile menu

Roathy committed:
Add mobile menu styling

Roathy committed:
Remove js from html and add scripts.js for js code

Roathy committed:
Add background image

Roathy committed:
Refactor code in two different files one for the mobile styling one for the medium size to big screen devices

Roathy committed:
Create folder for js file for better organization

Roathy committed:
Remove class .column

Remove unnecessary divs and containers

Add class to style cards in reverse direction

Roathy committed:
Move/delete files for better organization

Roathy committed:
Merge branch 'feature/mobile-menu' of https://github.com/Roathy/-Portfolio- into feature/mobile-menu

Roathy committed:
Hide hamburger button

Roathy committed:
Delete duplicated code for navbar

Add button to toggle menu

Roathy committed:
Delete scripts.js file replace by nav.js

Create navigation menu for mobile

Roathy committed:
Remove span container

Roathy committed:
Fix background color and border radious for contact section

Roathy committed:
Fix positioning of styles on proper classes

Roathy committed:
Fix background color and padding

Roathy committed:
Fix stylelint errors

Roathy committed:
Fix JS linters errors

Roathy committed:
Fix ESLint errors

Roathy committed:
Add onclick event to elements in menu

Roathy committed:
Fix font size for elements in menu

Roathy committed:
Add event listener function to close menu when clicked on link

Roathy committed:
Remove event listeners from html

Roathy committed:
Add event listener to elements using JS

Roathy committed:
Fix ESLint errors

emmanuellekamwa reviewed (approved):
**STATUS: APPROVED 🍾✔🎉**
Hello @Roathy 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Good commit messages.
- Descriptive PR.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
